pkg/sdn/plugin/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -0,0 +1,511 @@
+// +build linux
```

Comment:
So, the directory name ends up being the Go binary name, but that name is never seen outside of the build tree because we rename it when it gets installed? Why does it matter what it's called in the build tree then? (Oh, does it end up getting copied into some "output" directory with other unrelated things? Meh.)

I think just "cni-plugin" is probably sufficiently unambiguous. (Although on the flip side, "host-local" is quite mysterious and we're outputting that to the build output dir...)